_inc/client/plans/single-product-backup.jsx

```diff
@@ -36,3 +81,85 @@ export function PlanRadioButton( {
 		</label>
 	);
 }
+
+// eslint-disable-next-line no-unused-vars
+class SingleProductBackupBody extends React.Component {
```

Comment:
Feels odd that we have so many components in this same file. I'd suggest keeping one component per file for consistency and readability.